docs/source/reference/documentation.rst

```diff
@@ -0,0 +1,18 @@
+.. ...........................................................................
+.. © Copyright IBM Corporation 2024                                          .
+.. ...........................................................................
+
+=============
+Documentation
+=============
+
+In addition to the common reference material included in Helpful Links,
```

Comment:
should there be a link to "helpful links" here?

Reply:
Yes and no :) , I'll see about adding a link but the reason for this RST is so we can generate doc locally and ensure it all works before the full doc site ingests our doc, you can see it is linked by the parent doc site. 

<img width="1147" alt="image" src="https://github.com/ansible-collections/ibm_zos_core/assets/25803172/640a0ded-100e-4423-9353-eda1cfeaedf5">

Reply:
After looking into this, we would need to do something like

```
.. _Helpful Links:
   [helpful_links.html](https://ibm.github.io/z_ansible_collections_doc/reference/helpful_links.html)
```

As opposed to the below since this source is owned by the common docs, thus we have to maintain the link and there is really no benefit here because the doc site does not use our content, we only have this in our source so that we can generate doc without error. 
```
.. _Helpful Links:
   helpful_links.html
```